src/views/Wrap/Wrap.tsx

```diff
@@ -12,14 +12,14 @@ import {
   Link,
   MenuItem,
   OutlinedInput,
-  Paper,
+  // Paper,
   Select,
   SvgIcon,
   Typography,
   Zoom,
 } from "@material-ui/core";
 import { Skeleton } from "@material-ui/lab";
-import { Metric, MetricCollection } from "@olympusdao/component-library";
+import { Metric, MetricCollection, Paper } from "@olympusdao/component-library";
 import { DataRow } from "@olympusdao/component-library";
```

Comment:
This should all be one import
```suggestion
import { DataRow, Metric, MetricCollection, Paper } from "@olympusdao/component-library";
```

Reply:
fixed